Error when creating new analog project with NX

### Please provide the environment you discovered this bug in.

MacOS Arm M1
NX 22.1.1


### Which area/package is the issue in?

create-analog

### Description

When I try to create a new nx project with analog app:
npx create-nx-workspace@latest --preset=@analogjs/platform --package-manager=pnpm

I have this error after completing the first steps. (See errors)



### Please provide the exception or error you saw

```true
Fetching @nx/angular...
Fetching @angular-devkit/build-angular...

 NX   Cannot find module '@angular-devkit/core
 NX   Failed to apply preset: @analogjs/platform
```

### Other information

_No response_

### I would be willing to submit a PR to fix this issue

- [ ] Yes
- [ ] No

--- a/packages/nx-plugin/src/generators/app/files/template-angular/src/app/app.config.ts__template__
+++ b/packages/nx-plugin/src/generators/app/files/template-angular/src/app/app.config.ts__template__
@@ -1,6 +1,6 @@
 import {
   ApplicationConfig,
-  <% if (majorAngularVersion < 21) { %>provideZoneChangeDetection,<% } %>
+  <% if (majorAngularVersion < 21) { %>provideZonelessChangeDetection,<% } %>
   <% if (majorAngularVersion > 19) { %>provideBrowserGlobalErrorListeners,<% } %>
 } from '@angular/core';
 import {
@@ -13,7 +13,7 @@ import { provideFileRouter, requestContextInterceptor } from '@analogjs/router';
 
 export const appConfig: ApplicationConfig = {
   providers: [<% if (majorAngularVersion > 19) { %>provideBrowserGlobalErrorListeners(),<% } %>
-    <% if (majorAngularVersion < 21) { %>provideZoneChangeDetection({ eventCoalescing: true }),<% } %>
+    <% if (majorAngularVersion < 21) { %>provideZonelessChangeDetection(),<% } %>
     provideFileRouter(),
     provideClientHydration(),
     provideHttpClient(

--- a/packages/nx-plugin/src/generators/preset/generator.ts
+++ b/packages/nx-plugin/src/generators/preset/generator.ts
@@ -3,7 +3,8 @@ import { PresetGeneratorSchema } from './schema';
 
 export default async function (tree: Tree, options: PresetGeneratorSchema) {
   ensurePackage('@nx/angular', NX_VERSION);
-  ensurePackage('@angular-devkit/build-angular', 'latest');
+  ensurePackage('@angular-devkit/core', 'latest');
+  ensurePackage('rxjs', 'latest');
 
   return await import('../app/generator').then(({ appGenerator }) =>
     appGenerator(tree, options),

